server/testdata/endpoints/08_couper.hcl

```diff
@@ -0,0 +1,13 @@
+server "api" {
+  error_file = "./../integration/server_error.html"
+
+  endpoint "/pdf" {
+    request "pdf" {
+      url = "${env.COUPER_TEST_BACKEND_ADDR}/pdf"
+    }
+
+    response {
+      body = backend_responses.pdf.body
```

Comment:
A test with combination of proxy and request simultaneous body readout would be a nice addition.